src/main/Maintainer.cpp

```diff
@@ -3,16 +3,37 @@
 // of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
 
 #include "main/Maintainer.h"
+#include "main/Config.h"
 #include "main/ExternalQueue.h"
 #include "util/Logging.h"
 
 namespace stellar
 {
 
-Maintainer::Maintainer(Application& app) : mApp{app}
+Maintainer::Maintainer(Application& app) : mApp{app}, mTimer{mApp}
 {
 }
 
+void
+Maintainer::start()
+{
+    scheduleMaintenance();
```

Comment:
we could put a condition that `AUTOMATIC_MAINTENANCE_PERIOD > 0` to disable this